styles.css

```diff
@@ -0,0 +1,5 @@
+#image{
+    display:block;
+    max-width:100%;
+    text-align:center;
+}
```

Comment:
I would recommend using less spacing instead of three make it two for CSS best practice 😄